tox.ini

```diff
@@ -42,11 +38,9 @@ commands =
 
 [testenv:gettext]
 basepython = python3.9
-skip_install = true
```

Comment:
I'm surprised this was working. I suspect it may be because of sharing the `envdir` between `gettext` and `docs` -- if you already ran `tox -e docs`, then the venv would be set up properly such that you don't need to install `requirements.txt`/qiskit_metal too.

But I couldn't confirm this because I can't get this to build on Apple Silicon.